pkg/vm/engine/disttae/table.go

```diff
@@ -77,6 +78,61 @@ func (tbl *table) Stats(ctx context.Context) (int32, int64, error) {
 	return int32(totalBlockCnt), rows, nil
 }
 
+func (tbl *table) Rows(ctx context.Context) (int64, error) {
```

Comment:
I don't think it needs to be handled that way here. Why not simply try to add the number of rows in the block. What is the meaning of the exact value obtained with such an overhead?